be/test/exec/vectorized/chunks_sorter_test.cpp

```diff
@@ -1,976 +0,0 @@
-// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
why delete it ?

Reply:
there are some conflict in this file. so I remove this test